ops/jenkins/bfd-server-load-build/build-push.sh

```diff
@@ -1,8 +1,14 @@
 #!/usr/bin/env bash
 set -eou pipefail
 
+# SCRIPT_DIR will return the directory where this script is located, and CONTEXT_DIR will be the
+# directory of the Docker build context (locust_tests). This way, this script can be called from
+# _any_ directory and there will be no issues
+SCRIPT_DIR="$(readlink -f "$(dirname "${BASH_SOURCE[0]}")")"
+CONTEXT_DIR="$(cd "$SCRIPT_DIR"; cd ../../../apps/utils/locust_tests/; pwd)"
+
```

Comment:
I ❤️ that you're doing a little more in an attempt to standardize here. Do you think it would be a little clearere if we used dynamic, absolute paths rather than the relative paths here? Recently, I've been using `git` to determine the absolute path of the repository root, and constructing absolute paths from there. What do you think of _this_?
```suggestion
SCRIPT_DIR="$(readlink -f "$(dirname "${BASH_SOURCE[0]}")")"
REPO_ROOT="$(git rev-parse --show-toplevel)"
CONTEXT_DIR="${REPO_ROOT}/apps/utils/locust_tests/"

```